docs/os/contributing/index.md

```diff
@@ -29,7 +29,16 @@ If you're building a version of RancherOS used for development and not for a rel
 
 ### Testing
 
-Run `make integration-tests` to run the all integration tests in a container, or `./scripts/integration-tests` to run them outside a container (they use QEMU to test the OS.)
+Run `make integration-tests` to run the all integration tests in a container, or 
```

Comment:
We don't currently do line breaks in docs. If we switch over to this it's probably better to do it all at once rather than being inconsistent.